src/components/util/dragger.js

```diff
@@ -39,8 +39,12 @@ export function createDragger(fn, dragPreview) {
     // (2) setup drag listeners
 
     // attach drag + cleanup event
-    document.addEventListener('dragover', onDrag);
+    // we need to do this to make sure we track cursor
+    // movements before we reach other drag event handlers,
+    // e.g. in child containers.
+    document.addEventListener('dragover', onDrag, true);
```

Comment:
What do we want to accomplish by this? 

I'd assume a good dragging integration does two things:

* It only listens to `dragover*` events once it _activated_ for them, i.e. through `dragstart`
* Hence we want to prevent `dragstart` from reaching these applications (so they don't know dragging is in progress)

What do you think about that?

Reply:
`dragstart` is also the place where we want to [remove the ghost image](https://stackoverflow.com/questions/38655964/how-to-remove-dragghost-image), shown in [Firefox for example](https://github.com/camunda/web-modeler/pull/6222#issuecomment-1733144846).

Reply:
Never mind [this comment](https://github.com/bpmn-io/properties-panel/pull/292#discussion_r1335862742). I verified that dragover is what we're reacting to as we drop files.

Reply:
I recognize that we [already remove the Ghost image](https://github.com/bpmn-io/properties-panel/commit/9c027f2a28bb4ed9137a43d3dc9da9c89f759bcb#diff-73924921527587d6fc507d4c250f64ba3f5b66c197c18ba15019b269e763784fL36). It still seems to show in Firefox.

Reply:
> `dragstart` is also the place where we want to [remove the ghost image](https://stackoverflow.com/questions/38655964/how-to-remove-dragghost-image), shown in [Firefox for example](https://github.com/camunda/web-modeler/pull/6222#issuecomment-1733144846).

I need to have a second look at this, as this should have been build-in already, but it doesn't work anymore.

Reply:
[It is built-in](https://github.com/bpmn-io/properties-panel/pull/292#discussion_r1335879207), but seems to not work on Firefox. Let's regard it as an edge case.